lib/hammer_cli_foreman/hostgroup.rb

```diff
@@ -14,6 +14,14 @@ def self.included(base)
       base.option "--parent", "PARENT_NAME",  _("Name of parent hostgroup")
       base.option "--puppet-classes", "PUPPET_CLASS_NAMES", "",
         :format => HammerCLI::Options::Normalizers::List.new
+      base.option "--root-pass", "ROOT_PASSWORD",  _("Root password")
+      base.option "--ask-root-pass", "ASK_ROOT_PW", "",
+        :format => HammerCLI::Options::Normalizers::Bool.new
+    end
+
+    def self.ask_password
+      prompt = _("Enter the root password for the host group:")
```

Comment:
Unless `ask` adds it for you, you might need to put the space after the extracted string instead, e.g. `_("Enter:") + " "` so the UI's correctly spaced.